tests/integration/test_small_deployment_upgrades_edge_channel.py

```diff
@@ -76,74 +75,14 @@ async def test_deploy_latest_from_channel(ops_test: OpsTest) -> None:
     await ops_test.model.wait_for_idle(
         apps=[TLS_CERTIFICATES_APP_NAME, APP_NAME],
         status="active",
-        timeout=1400,
+        timeout=3600,
```

Comment:
Why are we increasing this timeout by this much? I'm afraid we hide issues that are visible to the user if we do this.

Reply:
I've noticed the upgrade is taking longer than 20 mins, including in my own workstation. It eventually works, it just takes longer.